resolvers/npm_installer/flag.rs

```diff
@@ -1,182 +1,220 @@
 // Copyright 2018-2025 the Deno authors. MIT license.
 
-use std::path::PathBuf;
-use std::sync::Arc;
-use std::time::Duration;
+pub use inner::LaxSingleProcessFsFlag;
+pub use inner::LaxSingleProcessFsFlagSys;
 
-use deno_unsync::sync::AtomicFlag;
-use sys_traits::FsFileLock;
-use sys_traits::FsMetadataValue;
+#[cfg(not(target_arch = "wasm32"))]
+mod inner {
```

Comment:
The diff below here is just putting the file into a module, then at the end of the file it's creating a wasm version of this, which is a no-op.